go/data/query.go

```diff
@@ -45,6 +45,7 @@ type (
 		QueryTime, LockTime    float64
 		RowsSent, RowsExamined int
 		Timestamp              int64
+		UsageCount             int
```

Comment:
this is not part of the slow query log, right? maybe we can move it to above the comment ☝️